src/include/homeobject/blob_manager.hpp

```diff
@@ -4,13 +4,14 @@
 #include <string>
 
 #include <sisl/fds/buffer.hpp>
+#include <homestore/blk.h>
```

Comment:
We shouldn't need to add anything to this header except the ENUM added. Can all be moved into hs_homeobject.hpp right?

Reply:
Ack